Formula/p11-kit.rb

```diff
@@ -39,11 +40,10 @@ def install
 
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
-                          "--disable-trust-module",
                           "--prefix=#{prefix}",
                           "--sysconfdir=#{etc}",
                           "--with-module-config=#{etc}/pkcs11/modules",
-                          "--without-libtasn1"
+                          "--with-trust-paths=#{etc}/ca-certificates/cert.pem"
```

Comment:
Shouldn't this have a `depends_on "ca-certificates"` then?

Reply:
The `/home/linuxbrew/.linuxbrew/etc/ca-certificates/cert.pem` is present on my system even though the ca-certificates brew is not installed

Reply:
You probably had it installed previously and have since uninstalled it. `etc` isn't cleaned up after a `brew uninstall`.

Reply:
You are right. Let me add the dependency.